apps/meteor/ee/app/livechat-enterprise/server/settings.ts

```diff
@@ -157,6 +157,32 @@ export const createSettings = async (): Promise<void> => {
 		});
 	});
 
+	await settingsRegistry.add('Livechat_AdditionalWidgetScripts', '', {
+		type: 'string',
```

Comment:
We should probably use a `type: 'code', code: 'javascript'` in this case

Reply:
Well, its a CSV list, more of a string than node no ?

Reply:
Indeed, now I've understood what the setting adds to the widget